packages/frontend/src/view/pages/home/HomePage.tsx

```diff
@@ -87,12 +85,6 @@ function HomePage(props: HomePageProps) {
               className="xl:hidden"
             />
           )}
-          {lastTutorial && (
```

Comment:
lets also change MAX_TUTORIALS to 2 then on line 76 remove the "- 1" part and change 48 lane to `const showViewAllTutorials = props.tutorials.length > MAX_TUTORIALS`